src/snippet.rs

```diff
@@ -44,6 +48,14 @@ impl<'a> Group<'a> {
         self
     }
 
+    /// Set the primary [`Level`] for this [`Group`]. If not specified, the
+    /// [`Level`] of the first [`Title`] in the [`Group`] will be used, if no
+    /// [`Title`] is present, it will default to [`Level::ERROR`].
```

Comment:
This needs a blank line after the first sentence so rustdoc can use that for the summary, rather than the whole block of text